pkg/api/bytes.go

```diff
@@ -30,6 +30,11 @@ func (s *server) bytesUploadHandler(w http.ResponseWriter, r *http.Request) {
 		jsonhttp.InternalServerError(w, "cannot get or create tag")
 		return
 	}
+	w.Header().Set(SwarmTagUidHeader, fmt.Sprint(tag.Uid))
+	w.WriteHeader(http.StatusContinue)
```

Comment:
Why setting the status to Continue as at the end it is written OK?

Status should be written only once.

Reply:
AFAIK `1xx` status codes can be sent before the final response code has been established. Even the golang built in http package actually is supposed to send a 100 continue while the body of the request is being read

Reply:
Hm. Ok. So, sending a 1xx status code before the end of the response is permitted by the HTTP protocol spec, but the golang http server doesn't support it. Except for the fact that it automatically sends a 100-continue automatically when reading the request body. But even if I write this extra header before the 100 continue is sent, the header doesn't get flushed with the 100 continue :/ I will continue testing this. Might be worth to open an issue on the golang repo for this

Reply:
closing this for now as with the built-in http server this is not possible.

https://github.com/golang/go/issues/40838